### PR TITLE
🇩🇪 updates 

### PR DESCRIPTION
**Pull request by dhinterlechner**
referenced many PR 😅

### CODE REVIEW

**Review by thisisjp — APPROVED** [review]

**Review by shadowsheep1 — APPROVED** [review]
LGTM

File: locales/de/index.yml [hunk 1]
```diff
@@ -1690,6 +1717,8 @@ wallet:
       WAITING_CONFIRMATION_EMAIL:
         title: "Warte auf eine Bestätigung per E-Mail"
         subtitle: "Es war nicht möglich, das Ergebnis der Zahlung zu überprüfen. Wenn sie erfolgreich war, wirst du in Kürze eine E-Mail erhalten. Sollte diese nicht ankommen, kontaktiere bitte den Support."
+        subtitle: "Du erhältst das Ergebnis deiner Zahlung mittels E-Mail an {{email}}."
```
Comment (shadowsheep1) [inline locales/de/index.yml:1720]: Here we have a duplicated key. @dhinterlechner could you please double check it?

Comment (dhinterlechner) [reply locales/de/index.yml]: forgot to delete the old one 🙈

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**